languages/eo.lua

```diff
@@ -1,4 +1,87 @@
+}
+
+SILE.settings:declare({ parameter = "languages.eo.use_raised_ordinal_suffix", type = "boolean", default = false })
+SILE.settings:declare({ parameter = "languages.eo.hyphen_before_ordinal_suffix", type = "boolean", default = false })
```

Comment:
SILE manual has a chapter about language-specific settings. Would need to be updated (in a separate "docs(manual)" commit likely.

Reply:
Also these settings have overly long names ^^

They both seem to refer to ordinals, so perhaps `languages.eo.ordinal.raisedsuffix` and `languages.eo.ordinal.hyphenbefore` ? (just an idea, to keep some namespacing and shorter flag names)